pwm1_vb_string: "[[14800.0, 0], [14700.0, -1.5], [14600.0, -2.0], [14500.0, -3.0]]"